docs/guides/quick_start_cheerio.ts

```diff
@@ -0,0 +1,20 @@
+import { CheerioCrawler, Dataset } from "@crawlee/cheerio";
+
+const crawler = new CheerioCrawler({
+    async requestHandler({ request, $, enqueueLinks }) {
+        const { url } = request;
+        // Extract HTML title of the page.
+        const title = $('title').text();
+        console.log(`Title of ${url}: ${title}`);
+        // Add URLs that match the provided pattern.
+        await enqueueLinks({
+            globs: ['https://www.iana.org/*'],
+        });
+        // Save extracted data to dataset.
+        await Dataset.pushData({ url, title });
+    },
+});
+// Choose the first URL to open.
```

Comment:
Really a nit but maybe some new lines would be nice?
```suggestion
});

// Choose the first URL to open.
```